utils/common/src/main/java/org/apache/brooklyn/util/ssh/BashCommands.java

```diff
@@ -671,6 +672,15 @@ public static String installJavaLatestOrWarn() {
     }
 
     /**
+     * Adds the PPA for OpenJDK for older JDK versions (7 and lower) required by some software (e.g. JBoss)
+     */
+    public static String addOpenJDKPPK(){
```

Comment:
Not sure that is the right was of doing this. I'm wondering if it would be better to handle this case by case, i.e. in the blueprints themselves